packages/windows-terminal.vm/tools/chocolateyinstall.ps1

```diff
@@ -54,6 +55,12 @@ try {
   $label = "Open Terminal here"
   $icon = "$executablePath"
   VM-Add-To-Right-Click-Menu -menuKey $toolName -menuLabel $label -command $command -menuIcon $icon -type "directory" -background
+
+  # Set windows terminal as the default terminal (effective only when OS Build >= 19045.3031)
```

Comment:
I am using `10.0.19045.0` (downloaded from the recommended link in FLARE-VM), so this approach does not work